web/src/index.js

```diff
@@ -17,7 +17,7 @@ let Main = () => {
           * New HUD work. HUD will eventually be promoted to the top-level component
           * and we'll delete App & friends
           */}
-        <Route exact path="/hud" component={HUD} />
+        <Route path="/hud" component={HUD} />
```

Comment:
attn @nicks this is tricky. With exact matching on for this top level any path like `/hud/foo` or `/hud/foo/bar` would be routed to missing route. It seems to work as expected with exact matching off, I think this is due to the way we're doing this nested "switch" "statement"?

_grumble grumble react router_

Reply:
this is the behavior we want tho? /hud/foo should go to the missing route. what am i missing?

Reply:
oh, nm, ya i understand, this is right